pub/refget-openapi.yaml

```diff
@@ -195,33 +190,34 @@ components:
       required:
         - alias
         - naming_authority
-    ServiceInfo:
-      type: object
-      description: Describes the capabilities of an API implementation
-      properties:
-        supported_api_versions:
-          type: array
-          items:
-            type: string
-            description: Supported versions by this API
-            example: 0.2.0
-        circular_supported:
-          type: boolean
-          description: Indicates if the service supports circular location queries
-        subsequence_limit:
-          type: integer
-          format: int64
-          nullable: true
-          description: Maximum length of sequence which may be requested using start and/or end query parameters
-        algorithms:
-          type: array
-          items:
-            type: string
-            description: An array of strings listing the digest algorithms that are supported
-            example:
-              - md5
-              - trunc512
-      required:
-        - supported_api_versions
-        - circular_supported
-        - algorithms
+    RefgetServiceInfo:
+      allOf:
+      - '$ref': https://raw.githubusercontent.com/ga4gh-discovery/ga4gh-service-info/develop/service-info.yaml#/components/schemas/Service
+      - type: object
+        description: Extended capabilities of refget service-info response object
+        required:
+          - refget
+        properties:
+          refget:
+            type: object
+            required:
+              - circular_supported
+              - algorithms
+            properties:
+              circular_supported:
+                type: boolean
+                description: Indicates if the service supports circular location queries
+              subsequence_limit:
+                type: integer
+                format: int64
+                nullable: true
+                description: Maximum length of sequence which may be requested using start and/or end query parameters
+              algorithms:
+                type: array
+                items:
+                  type: string
+                  description: An array of strings listing the digest algorithms that are supported
+                  example:
+                  - md5
+                  - ga4gh
+                  - trunc512
```

Comment:
Maybe we should remove `trunc512` from the example since it's deprecated

Reply:
Removed